PSCompatibilityCollector/Microsoft.PowerShell.CrossCompatibility/Collection/CompatibilityProfileCollector.cs

```diff
@@ -82,7 +82,7 @@ public CompatibilityProfileCollector Build(SMA.PowerShell pwsh)
             }
         }
 
-        private static readonly Version s_currentProfileSchemaVersion = new Version(1, 1);
+        private static readonly Version s_currentProfileSchemaVersion = new Version(1, 2);
```

Comment:
I am guessing the code works in such a way that and old schema (change only in minor version) is backwards compatible, i.e. old profiles still work?

Reply:
Exactly

Reply:
I've added a comment explaining this